edenia/antelope-tools
# Feat/add structured data for bps 1292

### Add structured data for BP Profile

### What does this PR do?

- Resolve #1292
- Add social media links and location into the ldjson
- Add P2P and API endpoints in the structured data
- Load non-square logos with an aspect ratio greater than 80%.
- Fix accessibility errors in nodes cards

### Steps to test

1. Run the project locally
2. Go to any BP Profile and check the structured data
